user-guide/live-traffic/web-sessions/general.md

```diff
@@ -0,0 +1,25 @@
+---
+title: General
+slug: web-sessions-general
+tags: web sessions, fiddler sessions, share sessions
+publish: true
+position: 10
+---
+
+# Web Sessions Section
```

Comment:
1) Consider using only one image with an arrow to the switch in the first image
2) Consider removing "Section" from the title
3) It feels like the **General** page does not have a clear purpose as it repeats text and images in the other two articles in this section. Consider removing it entirely.